### DISCUSSION TITLE
HOMEBREW_NO_AUTO_UPDATE=1 still updates?

### DISCUSSION THREAD

**Discussion by FreQRiDeR** [Homebrew/discussions#3394] [Everyday usage]
I entered ‘HOMEBREW_NO_AUTO_UPDATE=1 brew install mplayer’ and brew has been downloading new packages? Python 3.11 etc... It appears to be updating anyways? I have super slow internet and it’s taking forever! Multiple sessions over 3 days! I just want to grab a package! Lol! Granted, I am on Mojave, (unsupported) and it’s an old clone that hasn’t been updated in a long time? Is HOMEBREW_NO_AUTO_UPDATE superseded by incompatible packages/OS or am I doing something wrong? I’m afraid AUTO_UPDATE will break my brew install!

**Comment by gromgit** [comment]
The environment variable is called HOMEBREW_NO_AUTO_**UPDATE** for a reason: it stop automatic `brew update` runs before `brew install`.

And there's no HOMEBREW_NO_AUTO_**UPGRADE** environment variable, also for a reason: https://docs.brew.sh/FAQ#why-does-brew-upgrade-formula-or-brew-install-formula-also-upgrade-a-bunch-of-other-stuff

**Comment by FreQRiDeR** [comment]
Right ;)
